### PR TITLE
update tiers docs for membership APIs

### PR DESCRIPTION
**Pull request by presmihaylov**

### CODE REVIEW

File: _snippets/graphql/add-members-to-tier.mdx [hunk 1]
```diff
@@ -37,7 +37,7 @@ mutation addMembersToTier($input: AddMembersToTierInput!) {
         "companyId": "co_123"
       }
     ],
-    "tierId": "tier_123"
+    "tierIdentifier": { "tierId": "tier_123" }
```
Comment (mattvagni) [inline _snippets/graphql/add-members-to-tier.mdx:40]: Can you switch all of these to use externalId ? That is the most useful example and what the updateTenantTier and updateCompanTier mutations do.

File: _snippets/graphql/remove-members-from-tier.mdx [hunk 2]
```diff
@@ -25,8 +25,7 @@ mutation removeMembersFromTier($input: RemoveMembersFromTierInput!) {
       {
         "companyId": "co_123"
       }
-    ],
-    "tierId": "tier_123"
+    ]
```
Comment (mattvagni) [inline _snippets/graphql/remove-members-from-tier.mdx:28]: I don't understand - why are we no longer specifying which tiers I'm removing a member from? How does this work?

Comment (preslavmihaylov) [reply _snippets/graphql/remove-members-from-tier.mdx:28]: a tenant/company can only be part of a single tier. The ID of the entity is sufficient to remove it from the tier it's part of

File: _snippets/graphql/add-members-to-tier.mdx [hunk 3]
```diff
@@ -37,7 +37,7 @@ mutation addMembersToTier($input: AddMembersToTierInput!) {
         "companyId": "co_123"
       }
     ],
-    "tierId": "tier_123"
+    "tierIdentifier": { "externalId": "tier_123" }
```
Comment (mattvagni) [inline _snippets/graphql/add-members-to-tier.mdx:40]: `tier_123` could be confused with our tier ids. Can you use `XXX` which is what I think we use in the other externalId tier examples we have in the docs?

File: _snippets/graphql/remove-members-from-tier.mdx (hunk 2 above)
Comment (mattvagni) [reply _snippets/graphql/remove-members-from-tier.mdx:28]: Amazing - makes sense thanks.

File: _snippets/typescript-sdk/add-members-to-tier.mdx [hunk 4]
```diff
@@ -12,7 +12,7 @@ const res = await client.addMembersToTier({
       companyId: 'co_123',
     },
   ],
-  tierId: 'tier_123',
+  tierIdentifier: { externalId: 'tier_123' },
```
Comment (mattvagni) [inline _snippets/typescript-sdk/add-members-to-tier.mdx:15]: Same here - I think `XXX` would be better to not confuse this with our ids.

File: api-reference/graphql/tiers/add-members.mdx [hunk 5]
```diff
@@ -5,6 +5,8 @@ sidebarTitle: 'Add companies & tenants'
 
 You can add multiple tenants and companies to a tier in a single mutation.
 
+An entity (company or tenant) can only be part of a single tier.
```
Comment (mattvagni) [inline api-reference/graphql/tiers/add-members.mdx:8]:
```suggestion
Companies and tenants can only be in a single tier.
```

**Review by mattvagni — APPROVED** [review]

**Pull request merged**